test/html/gcweb/TEST-widgets-en.html

```diff
@@ -0,0 +1,58 @@
+<!DOCTYPE html>
+<!--[if lt IE 9]><html class="no-js lt-ie9" lang="en" dir="ltr"><![endif]-->
+<!--[if gt IE 8]><!-->
+<html class="no-js" lang="en" dir="ltr">
+<!--<![endif]-->
+
+<head>
+    <meta charset="utf-8">
+    <!-- Web Experience Toolkit (WET) / Boîte à outils de l'expérience Web (BOEW) wet-boew.github.io/wet-boew/License-en.html / wet-boew.github.io/wet-boew/Licence-fr.html -->
+    <title>Test widgets - Canada.ca</title>
+    <meta content="width=device-width,initial-scale=1" name="viewport">
+    <!-- Meta data -->
+    <meta name="description"
+        content="Web Experience Toolkit (WET) includes reusable components for building and maintaining innovative Web sites that are accessible, usable, and interoperable. These reusable components are open source software and free for use by departments and external Web communities">
+    <!-- Meta data-->
+    <!-- Load closure template scripts -->
+    <link rel="stylesheet" href="../cdts-styles.css">
+    <script src="../compiled/wet-en.js" data-cdts-setup='{
+        "cdnEnv": "localhost",
+        "base": {"cdnEnv":"localhost"},
+        "top": {"cdnEnv": "localhost"},
+        "preFooter": {"cdnEnv": "localhost"},
+        "footer": {"cdnEnv": "localhost"},
```

Comment:
Not really a big deal, everything works anyway, but might as well correct it in case this is used in a copy&paste in the future :)

```suggestion
        "cdnEnv": "localhost",
        "base": {},
        "top": {},
        "preFooter": {},
        "footer": {},
```